### DISCUSSION TITLE
JDemetra+ Universe Blog - Resources from the Webinar on R tools for JDemetra+

### DISCUSSION THREAD

**Discussion by giscus** [annasmyk/JD_Quarto_Blog#3] [General]
# JDemetra+ Universe Blog - Resources from the Webinar on R tools for JDemetra+

https://jdemetra-universe-blog.netlify.app/posts/resources_webinar_rjd/

**Comment by annasmyk** [comment]
Some people asked for the recording of the webinar. We are editing the video and will upload it on GitHub (same repo).

**Reply by annasmyk** [reply]
Video is available !
